lib/improver/tests/acceptance/test_nbhood.py

```diff
@@ -97,8 +113,12 @@ def test_masked_square_external(tmp_path):
     input_path = kgo_dir / "input.nc"
     mask_path = kgo_dir / "mask.nc"
     output_path = tmp_path / "output.nc"
-    args = ["probabilities", "square", input_path, output_path,
-            "--radius=20000", "--input_mask_filepath", mask_path]
+    args = [input_path, mask_path,
+            "--neighbourhood-output", "probabilities",
+            "--neighbourhood-shape", "square",
+            "--radii", "20000",
+            "--weighted-mode",
```

Comment:
`weighted-mode` wasn't in the original test. Why is it here?

Reply:
I meant to come back to this. I was interested in why it made no difference, but I think my interest has waned for now. I'll remove this.

Reply:
This makes no difference because the argument has literally no effect in the SquareNeighbourhood class.  Quoting the `__init__` docstring:
```
weighted_mode (boolean):
    This is included to allow a standard interface for both the
    square and circular neighbourhood plugins.
```